pkg/decoder/decoder.go

```diff
@@ -177,3 +179,11 @@ type UplinkFeatureButton interface {
 type UplinkFeatureResetReason interface {
 	GetResetReason() ResetReason
 }
+
+type UplinkFeatureRotationState interface {
+	GetRotationState() RotationState
+}
+
+type UplinkFeatureSequenceNumber interface {
+	GetSequenceNumber() int
```

Comment:
use an uint8 as source data indicates or uint

Reply:
I agree, but the uint8 might be an issue for future projects.

```suggestion
	GetSequenceNumber() uint
```